src/ini/hashmap/hashmap.h

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020 Joshua J Baker. All rights reserved.
```

Comment:
I would place hashmap implementation in the common dir.

Reply:
I made the change can you take a look